### PR TITLE
datetime docs

### PR DESCRIPTION
**Pull request by jebeck**
What it says on the tin. Hope these are useful! Already published to GH Pages, so if y'all want to edit them before merging, you should republish.

I dunno who to ping on this, perhaps @gniezen, @krystophv, and @cbwebdevelopment.

### CODE REVIEW

**Comment by darinkrauss** [comment]
Why don't all three of you,  @gniezen, @krystophv, and @cbwebdevelopment, review and approve. Whoever approves last has to merge into master! ;)  (Unless, of course, there is anything that needs to be addressed before it can be approved.)

File: datetime/diabetes-data.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+In the client, the guideline for handling diabetes data datetimes is quite simple: use the `time` field in the Tidepool data model, in combination with the user's configured timezone preferences.
+
+When using [Moment.js](https://momentjs.com/ 'Moment.js') to manipulate and/or display diabetes device datetimes using the `time` field in the Tidepool data model, in every case your usage should begin with `moment.utc(time).tz(timezone)`. For good examples of this, refer to [viz's datetime utilities](https://github.com/tidepool-org/viz/blob/master/src/utils/datetime.js '@tidepool/viz: datetime utilities').
+
+
+- - - - -
+
+[^a]: Not pathologically or maliciously badly. Just, you know, *poorly*.
+[^b]: Just kidding! We haven't actually seen this one yet at Tidepool. But it's the *only one*. Yes. *Really.*
```
Comment (gniezen) [inline datetime/diabetes-data.md]: footnotes `^b` and `^c` should be moved to `falsehoods.md`

**Review by gniezen — CHANGES_REQUESTED** [review]
Well done @jebeck on writing a very entertaining tutorial (as much as it could be ;) ) on datetimes at Tidepool.

File: datetime/diabetes-data.md (hunk 1 above)
Comment (jebeck) [reply datetime/diabetes-data.md]: I can make this small change on this branch if @darinkrauss or whoever adds me to the docs repo as an external contributor. Or feel free to edit the branch yourselves 🙂

Comment (clintonium-119) [reply datetime/diabetes-data.md]: Thanks, Jana.  I'll push these changes.  Nice job 👍

**Review by clintonium-119 — APPROVED** [review]
I've moved and published the 2 out-of-place footnotes as suggested in @gniezen's feedback.

LGTM

**Review by krystophv — APPROVED** [review]
LGTM 👍

**Review by gniezen — APPROVED** [review]

**Pull request merged**